yeonjae/programmers-2/피보나치수.js

```diff
@@ -0,0 +1,25 @@
+function solution(n) {
+  const result = [];
+
+  for (let i = 0; i <= n; i++) {
+    if (i == 0) {
+      result.push(0);
+    }
+
+    if (i == 1) {
+      result.push(1);
+    }
+
+    if (i >= 2) {
+      let sum = result[i - 2] + result[i - 1];
+      result.push(sum % 1234567);
+    }
+  }
+
+  return result[n] % 1234567;
+}
```

Comment:
```suggestion
function solution(n) {
  const result = [];

  for (let i = 0; i <= n; i++) {
    if (i <= 1) {
      result.push(i);
      continue;
    }
    const sum = result[i - 2] + result[i - 1];
    
    result.push(sum % 1234567);
  }

  return result[n] % 1234567;
}
```
- 풀이잘봤습니다~
- 음 ㅠㅠ 왜 오늘 문제들 모두 일치 연산자(===)가 아닌 동등(==)연산자를 쓰신 이유가 있으실까요? 평소에 잘하시다가 오늘 문제들 모두 동등 연산자를 쓰신 이유가 궁금하네요
- i == 0, i == 1은 i <= 1 로 처리가 가능합니다.
- 또한 continue를 사용해서 조건문을 최소화 할 수 있습니다 참고해주세요